onnxscript/tools/benchmark/export_model_test.py

```diff
@@ -135,6 +138,34 @@ def test_export_model_phi_cpu_dynamo_llama0(self):
         out = f.getvalue()
         self.assertIn(":repeat_time,", out)
 
+    @unittest.skipIf(not has_transformers(), reason="transformers missing")
+    @unittest.skipIf(torch_older_than("2.4"), reason="fails to export")
```

Comment:
I think we can improve the reason to say

```suggestion
    @unittest.skipIf(torch_older_than("2.4"), reason="Fails to export with torch<2.4")
```
```suggestion
    @unittest.skipIf(torch_older_than("2.4"), reason="fails to export")
```